### PR TITLE
Log execution of a distributed query

### PR DESCRIPTION
**Pull request by mwielgoszewski**
Meets the minimum requirements asked of #2164, by logging execution of distributed queries.

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

File: osquery/distributed/distributed.cpp [hunk 1]
```diff
@@ -120,6 +120,7 @@ void Distributed::addResult(const DistributedQueryResult& result) {
 
 Status Distributed::runQueries() {
   while (getPendingQueryCount() > 0) {
+    VLOG(1) << "Executing distributed query: " << query.query;
```
Comment (theopolis) [inline osquery/distributed/distributed.cpp]: `query` is not available here, move this below line 124. You may also want the name of the query too?
```cpp
VLOG(1) << "Executing distributed query (" << query.id << "): " << query.query;
```

Comment (mwielgoszewski) [inline osquery/distributed/distributed.cpp]: Doh, `dd`'d with the cursor in the wrong spot.

**Comment by zwass** [comment]
ok to test

**Comment by osqueryer** [comment]
Build origin/pr/2241/merge (Job: 2871) failed :-1:

**Comment by zwass** [comment]
test failure looks unrelated

**Pull request merged**